source/kernel/kernel.go

```diff
@@ -114,6 +114,7 @@ func parseVersion() (map[string]string, error) {
 	// Replace forbidden symbols
 	fullRegex := regexp.MustCompile("[^-A-Za-z0-9_.]")
 	full = fullRegex.ReplaceAllString(full, "_")
+	full = strings.Trim(full, "-_.")
```

Comment:
you need to add `strings` to the imports :)

Reply:
😭  I was a bit too quick on this one. Fixed.